src/modules/commander/PreflightCheck.cpp

```diff
@@ -572,7 +572,7 @@ static bool ekf2Check(orb_advert_t *mavlink_log_pub, bool optional, bool report_
 	param_get(param_find("COM_ARM_EKF_AB"), &test_limit);
 	if (fabsf(status.states[13]) > test_limit || fabsf(status.states[14]) > test_limit || fabsf(status.states[15]) > test_limit) {
 		if (report_fail) {
-			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: EKF HIGH IMU ACCEL BIAS");
+			mavlink_log_critical(mavlink_log_pub, "PREFLIGHT FAIL: EKF HIGH IMU ACCELEROMETER BIAS.");
```

Comment:
Change no longer needed. Text already improved in this more recent PR:
https://github.com/PX4/Firmware/pull/10586/files#diff-a2bcb238e25d8a97ab2aee48f54dbc48R582